config/dynamic.c

```diff
@@ -119,9 +120,6 @@ process_client_message(int conn_fd,
 	/*
 	 * TODO #61 Implement the functionalities to process the clients' request.
 	 *
-	 * Gatekeeper and Grantor: Listing the ARP and ND table.
-	 * This is important for network diagnosis.
-	 *
 	 * Grantor: Updating the policy GTs run.
 	 */
```

Comment:
The merged pull request #154 enabled the dynamic configuration block to update the policy of GT blocks. Qiaobin forgot to update the comment above then. Thus, this patch, in fact, closes issue #61. Besides dropping the whole comment above, update the description of this patch to close #61.